src/Features/CSharp/Portable/Completion/CompletionProviders/DeclarationNameCompletionProvider.DeclarationInfo.cs

```diff
@@ -59,8 +65,42 @@ internal struct NameDeclarationInfo
                 return default(NameDeclarationInfo);
             }
 
+            private static bool IsPossibleOutVariableDeclaration(SyntaxToken token, SemanticModel semanticModel, int position,
+                ITypeInferenceService typeInferenceService, CancellationToken cancellationToken, out NameDeclarationInfo result)
+            {
+                if (!token.IsKind(SyntaxKind.IdentifierToken) || !(token.Parent.IsKind(SyntaxKind.IdentifierName)))
+                {
+                    result = default;
+                    return false;
+                }
+
+                var argument = token.Parent.Parent as ArgumentSyntax // Foo(out var $$
+                    ?? token.Parent.Parent.Parent as ArgumentSyntax; // Foo(out var a$$
```

Comment:
not sure i like this.  We have an existing system to say "move to the left one token if i'm on an actual name".  i'd prefer we stick witth htat pattern.